src/cmd/test.go

```diff
@@ -21,6 +26,20 @@ var testCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
+		serviceName := ""
+		if _, err = os.Stat("service.yml"); err == nil {
+			serviceConfig := types.ServiceConfig{}
+			yamlFile, err := ioutil.ReadFile(path.Join(appDir, "service.yml"))
+			if err != nil {
+				panic(err)
+			}
+			err = yaml.Unmarshal(yamlFile, &serviceConfig)
+			serviceName = serviceConfig.Type
+			appDir = path.Join(appDir, "/..")
```

Comment:
Thoughts on renaming the initial `appDir` variable (set from the call to `os.Getwd`) to `currentDir` and updating this branching tree to the following:
```go
if _, err = os.Stat("service.yml"); err == nil {
  // Assign serviceName 
  // Assign appDir as the parent of the currenDir 
} else if _, err = os.Stat("application.yml"); err == nil {
  // Assign appDir as currentDir
} else {
  // Print error message
}
```
I think thats more clear than assigning the currentDir as the appDir and overwriting it when that is wrong